src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,34 @@
+//NAV BLOCK:
+
+.nav {
+
+  //ELEMENTS:
+  &__list {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    background-color: white;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  /* nav-bar links decoration: */
+  &__link {
+    display: flex;
+    align-items: center;
+    height: 60px;
```

Comment:
![image](https://user-images.githubusercontent.com/104728827/185793124-efb1932c-562d-4beb-9b39-8fea2634365e.png)
Try to not set a definite height size